Index: app/src/main/java/com/example/platform/SignUp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/platform/SignUp.java	(revision 6383b136fd3b575012b45fb16b629701ef02a3a2)
+++ app/src/main/java/com/example/platform/SignUp.java	(date 1620483058874)
@@ -13,4 +13,5 @@
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_sign_up);
     }
+
 }
\ No newline at end of file
